app/src/main/java/de/westnordost/streetcomplete/quests/diet_type/DietType.kt

```diff
@@ -0,0 +1,7 @@
+package de.westnordost.streetcomplete.quests.diet_type
+
+enum class DietType(val osmValue: String) {
```

Comment:
Under the name "diet type", I'd expect values such as "vegan", "halal", "vegetarian" etc.

Right now, no better name for the enum comes to my mind... HasDiet?

Reply:
`DietAvailability`?

Reply:
sounds good